src/scheduleBot_Manager.py

```diff
@@ -1,734 +1,420 @@
+                # 0 one
                 if tdelta.days == 0:
                     delta_minutes = tdelta.seconds // 60
                     delta_seconds = tdelta.seconds % 60
+
+                    # length of the lesson is 95 minutes.
+                    # if delta higher then 94 - there is still a break
                     if delta_minutes > 94:
-                        result = "Погоди, все ещё идет перемена (или же сейчас утро), *{}*, отдыхай\n/nextLesson — покажет тебе информацию о следующей паре".format(name)
+                        result = f"Зачекай, зараз немає пари. Мабуть, йде" \
+                                 f"перерва або все ще ранок, *{name}*, " \
+                                 f"відпочивай\n" \
+                                 f"/nextLesson — покаже тобі інформацію " \
+                                 f"про наступну пару"
                         return result
+
+                    # else - get current lesson from xml
+                    # and show delta as the time till the end of the lesson
                     curr_lesson = self.xls.get_current_lesson(
                         group,
                         self.timetable,
                         weekday,
-                        weeknum,
+                        week_num,
                         i + 1
                     )
-                    result  = "Сейчас идет _{} пара_:\n".format(i + 1)
+                    result = f"Наразі йде _{i + 1} пара_:\n"
                     result += curr_lesson
-                    result += "\nДо конца этой пары: *{} мин {} сек*".format(
-                        delta_minutes,
-                        delta_seconds
-                    )
+                    result += f"\nДо кінця цієї пари: *{delta_minutes} хв " \
+                              f"{delta_seconds} сек*"
                     if curr_lesson == '':
-                        result = "Сейчас идет _{} пара_:\nНо у тебя на ней *ничего* нет :)\n------------------------------\nВремя до конца этой пары: *{} мин {} сек*\n------------------------------\n/nextLesson — покажет тебе информацию о следующей паре".format(i + 1, delta_minutes, delta_seconds)
+                        result = f"Наразі йде _{i + 1} пара_:\n" \
+                                 f"Але у тебе на ній *нічого* немає :)\n" \
+                                 f"------------------------------\n" \
+                                 f"Час до кінця цієї пари: " \
+                                 f"*{delta_minutes} хв {delta_seconds} " \
+                                 f"сек*\n" \
+                                 f"------------------------------\n" \
+                                 f"/nextLesson — покаже тобі інформацію " \
+                                 f"про наступну пару"
                     return result
         except Exception as e:
-            print(e)
-            result = "Произошла ошибка, извините...".format(name)
+            # I give it 1/1000 that there will be an exception
```

Comment:
Раз уж такая пьянка то и я на инглиш перейду
Even if it's 1/1000 it's still a great practice to catch an exceptions. But in that case I think the most dangerous part is `get_current_lesson()` and there is no need to put usual math operations in try/catch. And also putting that much code in one try/except is not great as well, because you can catch stuff you don't expect and sometime you notice that too late